src/main/java/uk/gov/pay/directdebit/webhook/gocardless/services/WebhookGoCardlessService.java

```diff
@@ -14,14 +14,18 @@
 import java.util.List;
 
 public class WebhookGoCardlessService {
+
     private static final Logger LOGGER = PayLoggerFactory.getLogger(WebhookGoCardlessService.class);
-    private final TransactionService transactionService;
+
     private final GoCardlessService goCardlessService;
+    private final GoCardlessPaymentHandler goCardlessPaymentHandler;
+    private final GoCardlessMandateHandler goCardlessMandateHandler;
 
     @Inject
     public WebhookGoCardlessService(GoCardlessService goCardlessService, TransactionService transactionService) {
         this.goCardlessService = goCardlessService;
-        this.transactionService = transactionService;
+        goCardlessPaymentHandler = new GoCardlessPaymentHandler(transactionService, goCardlessService);
+        goCardlessMandateHandler = new GoCardlessMandateHandler(transactionService, goCardlessService);
```

Comment:
I need to pass in the event to the handler in my PRs, so I will probably move this back to the switch statement, but it's all right for now :)